api/v1alpha/example_client.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Copyright 2016 Author Josef Boudnik - josef.boudnik@seznam.cz  
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+istributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from grpc.beta import implementations
+import api_pb2
+
+host = "localhost"
+port = 15441
+TIMEOUT = 3
+
+def run():
+        channel = implementations.insecure_channel(host,port)
+        connect = api_pb2.beta_create_PublicAPI_stub(channel)
+        channel = None
```

Comment:
Use: 
```
del channel
```
in this place. Setting value for name to `None` and then not using it - looks ugly and will not pass through any static code checker.
Btw. it looks like this library does not have any support to "close connection" because as a transport method there is HTTP used, so stateless/long living connection less method of communication, so there is no "connection closing".